src/profile-logic/merge-compare.js

```diff
@@ -769,6 +769,7 @@ function combineSamplesDiffing(
       // of eventDelay/responsiveness don't mean anything.
       newSamples.eventDelay.push(null);
       newSamples.time.push(samples1.time[i]);
+      // TODO: Figure out a way to diff CPU usage numbers.
```

Comment:
I guess we need to deal with this at some point but this is out of this PR's scope. I can file a follow-up if you want?

Reply:
Yes, please file a follow-up and put the bug number in the comment. Thanks for thinking of that!
I don't have a good solution for now...